test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -181,16 +192,25 @@ public SignatureTrustAndValidityVerificationProviderTests(SigningTestFixture fix
                 LogLevel.Warning,
                 setting);
 
-            Assert.Empty(matchingIssues);
+            Assert.Equal(2, matchingIssues.Count);
```

Comment:
Could we have a helper method to handle these two message assertions for all of 4 (?) of these tests? We should at least have constants for the two messages.